include/deal.II/lac/arpack_solver.h

```diff
@@ -162,6 +162,14 @@ class ArpackSolver : public Subscriptor
   void set_initial_vector(const VectorType &vec);
 
   /**
+   * Set real @p sigmar and complex @p sigmai parts of the shift for
+   * shift-and-invert spectral transformation.
+   *
+   * If this function is not called, the shift is assumed to be zero.
+   */
+  void set_shift(const double sigmar, const double sigmai = 0);
```

Comment:
it looks like it does this correctly, at least on my `clang`. Do you want me to change to `std::complex` here?